css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
+
+import (
+	"encoding/json"
+
+	"sourcegraph.com/sourcegraph/srclib/graph"
+)
+
+func init() {
+	graph.RegisterMakeDefFormatter("Dir", newDefFormatter)
+}
+
+// DefData should be kept in sync with the def 'Data' field emitted by the
+// basic grapher.
+type DefData struct {
+	Name      string
+	Keyword   string
+	Type      string
+	Kind      string
+	Separator string
+}
+
+func newDefFormatter(s *graph.Def) graph.DefFormatter {
+	var si DefData
+	if len(s.Data) > 0 {
+		if err := json.Unmarshal(s.Data, &si); err != nil {
+			panic("unmarshal CSS def data: " + err.Error())
+		}
+	}
+	return defFormatter{s, &si}
+}
+
+type defFormatter struct {
+	def  *graph.Def
+	data *DefData
+}
+
+func (f defFormatter) Language() string { return "CSS" }
+
+func (f defFormatter) DefKeyword() string { return f.data.Keyword }
```

Comment:
I think that `Keyword`, `Kind`, `Separator`, and `Type` may be omitted in the `Data` section. I'd day that all these fields should be empty string for CSS